_deployment/tests/twitter.sh

```diff
@@ -4,26 +4,29 @@
 check_twitter () {
   handles="$(git diff origin/master...HEAD ../../_data | grep "^+[[:space:]]*twitter:" | cut -c15-)"
 
-  if [ -z "$handles" ]; then
+  if [[ ! "$handles" =~ [^[:space:]] ]]; then
     echo "No Twitter handles found."
     exit 0
   fi
 
   gem i twitter --no-post-install-message --no-suggestions --minimal-deps --no-verbose -N -q --silent
 
+  res=0
   # Loop through all Twitter handles
-  echo "${handles}" | while IFS= read -r handle; do
+  for handle in $handles; do
```

Comment:
Is this line change needed?

Reply:
Yes, it is - the pipe symbol "|" starts a new sub-shell, and any assignments to variables in there do _not_ propagate to the outer shell.